R/system.R

```diff
@@ -469,7 +469,8 @@ sql_glue_transformer_internal <- function(expr, envir, bigquery=FALSE) {
     }
   }
 
-  if (is.null(val)) { # NULL in R is same as empty vector. Print empty string.
+  if (length(val) == 0) { # Empty vector case. NULL in R is same as empty vector, but since is.null(as.character(c(0))) returns FALSE, this has to be done my length().
```

Comment:
Do you mean "by length()" by "my length()"?

Reply:
Fixed it. Thanks!